### PR TITLE
Workbench: ISIS Reflectometry pause button is disabled on opening

### LINKED ISSUE #26394: ISIS Reflectometry: Pause button on runs table is enabled on opening
### Expected behavior
It should be disabled

### Actual behavior
It's enabled

### Steps to reproduce the behavior
Open workbench->Interfaces->ISIS Reflectometry
The pause button should look like this: 
image

However, it does not.

### Platforms affected
All

### PR DESCRIPTION
**Pull request by PhilColebrooke**
**Description of work.**
The pause button in the ISIS Reflectometry interface is now disabled when the interface is first opened, and becomes enabled when processing is started.

**To test:**
1. Open Workbench -> Interfaces -> ISIS Reflectometry.
2. The pause button should be disabled.
3. Begin processing. Example numbers: Run number 13460 Angle 0.05, Run number 13463 Angle 0.05. (set instrument to INTER).
4. The pause button should become enabled. Try pausing and unpausing to ensure the button becomes enabled and disabled correctly.

Fixes #26394 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by Pasarus — CHANGES_REQUESTED** [review]
Could you add a test to ensure that it is disabled on start?

**Review by Pasarus — DISMISSED** [review]

**Comment by gemmaguest** [comment]
I think it might be better to do this via the presenter so we can test it. I'll come over to chat about it.

**Review by gemmaguest — APPROVED** [review]
Thanks for the changes. Looks good now and behaves as expected :+1: :+1:. Happy to merge when tests pass.

The code has been written in a way to make testing easy but tests don't exist yet for the main window and it's beyond the scope of this PR to add them, so we agreed that this would be done later as part of #26025

File: qt/scientific_interfaces/ISISReflectometry/GUI/Batch/BatchPresenter.h [hunk 1]
```diff
@@ -65,6 +65,7 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL BatchPresenter
   void notifySettingsChanged() override;
   void anyBatchAutoreductionResumed() override;
   void anyBatchAutoreductionPaused() override;
+  void reductionPaused() override;
```
Comment (DanNixon) [inline qt/scientific_interfaces/ISISReflectometry/GUI/Batch/BatchPresenter.h]: @gemmaguest I know it's unrelated and low priority, but the naming of these functions imply they are getters (i.e. `reductionPaused()` would return a `bool` indicating if the reduction was paused).

A better name might be `notifyReductionPaused()`, etc.

**Pull request merged**